etl/tests/test_transform.py

```diff
@@ -17,13 +18,24 @@ def azure_reader(populated_azure_emulator: transform.AzureCoordinates) -> transf
 
 def test_reader(local_reader: transform.LocalExtractReader) -> None:
     output = list(local_reader.extracted_rows())
+
     output = sorted(output, key=lambda row: row['BUILDER'])
     unique_builders = {row['BUILDER'] for row in output}
+
     assert len(output) == 14
     assert output[0]['BUILDER'] == '11W2D00606'
     assert len(unique_builders) == 14
 
 
+def test_reader_sorted_by_eval_id(local_reader: transform.LocalExtractReader) -> None:
+    output = list(local_reader.extracted_rows())
+
+    assert all(
+        typing.cast(str, current_row.get('EVAL_ID')) <= typing.cast(str, next_row.get('EVAL_ID'))
```

Comment:
`current_row.get('EVAL_ID')` isn't already a string?

Reply:
It is, but mypy doesn't know that. `typing.cast` doesn't actually do anything to do the data, it just returns exactly what you passed it, but it does inform the type system.